docs/schema_change_protocol.md

```diff
@@ -14,37 +14,38 @@ Author: Phil Snyder
 * d. Run `json_to_parquet` workflow for the relevant JSON dataset / Glue table
     * If the JSON dataset is able to be successfully loaded into a DynamicFrame and exported and read as parquet, our new schema is compatible with the older schemas
     * Otherwise, the schemas are not compatible
 
 ## 3. IF new schema is compatible with old schemas:
 
-* a. Deactivate the initial trigger of each study's `json_to_parquet` workflow
-    * Once we reactivate this trigger, no new data will be incorporated into any parquet dataset until all files from the soon-to-be-updated JSON dataset / Glue table have finished reprocessing. This is due to our `json_to_parquet` workflow having a max concurrency of 1.
-* b. Update the appVersion to dataset mapping to include the appVersion which will use the new schema.
-* c. Archive existing and relevant parquet datasets.
+* a. Update the appVersion to dataset mapping to include the appVersion which will use the new schema.
+* b. Archive existing and relevant parquet datasets.
     * This entails moving relevant datasets in `s3://app/study/parquet/` to an archive folder `s3://app/study/parquet/archive/{dataset}_{version}_{update_number}/` . For example, if the `taskData_v2` dataset has already undergone three compatible schema updates, then we move all `taskData_v2_*` parquet datasets to `s3://app/study/parquet/archive/taskData_v2_4/` for each study. `taskData_v2_1`, `taskData_v2_2`, and `taskData_v2_3` already exist in this archive folder because we have performed this step three times already in response to previous schema changes.
-* d. Update glue table definition in prod with new schema for each study.
-* e. Reset `json_to_parquet` job bookmark for the relevant `json_to_parquet` job.
+* c. Create or update the following AWS resources in the stack:
+    * Update the Glue table schema for each affected study.
+* d. Deploy new stack.
+* e. Reset `json_to_parquet` job bookmark for this `{dataset} ` and`{dataset_version}`'s json_to_parquet job.
     * This will cause this job's next run to process every file in its respective JSON dataset.
-* f. Reactivate the initial trigger of each study's `json_to_parquet` workflow.
+* f. Delete existing parquet dataset. This dataset will be written again after the `json_to_parquet` job completes its run through the entire JSON dataset.
+
```

Comment:
Maybe want to archive this at least temporarily instead? So users can continue to have some data access while the data is regenerated.